{"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"waiterFailed","kind":"identifier"},{"text":": ","kind":"text"},{"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError","text":"ClientError","kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore9AWSClientC11ClientErrorV"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}],"platforms":["macOS"]}],"kind":"declarations"}],"identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError\/waiterFailed","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Waiter failed, but without an error. ie a successful api call was an error"}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError"]]},"kind":"symbol","metadata":{"symbolKind":"property","title":"waiterFailed","externalID":"s:8SotoCore9AWSClientC11ClientErrorV12waiterFailedAEvpZ","fragments":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"waiterFailed","kind":"identifier"},{"kind":"text","text":": "},{"text":"ClientError","kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore9AWSClientC11ClientErrorV"}],"modules":[{"name":"SotoCore"}],"roleHeading":"Type Property","role":"symbol"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/sotocore\/awsclient\/clienterror\/waiterfailed"]}],"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSClient":{"role":"symbol","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"AWSClient"}],"type":"topic","title":"AWSClient","fragments":[{"kind":"keyword","text":"class"},{"text":" ","kind":"text"},{"text":"AWSClient","kind":"identifier"}],"abstract":[{"type":"text","text":"Client managing communication with AWS services"}],"url":"\/documentation\/sotocore\/awsclient","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSClient/ClientError/waiterFailed":{"title":"waiterFailed","abstract":[{"type":"text","text":"Waiter failed, but without an error. ie a successful api call was an error"}],"fragments":[{"text":"static","kind":"keyword"},{"kind":"text","text":" "},{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"waiterFailed"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:8SotoCore9AWSClientC11ClientErrorV","kind":"typeIdentifier","text":"ClientError"}],"type":"topic","kind":"symbol","role":"symbol","url":"\/documentation\/sotocore\/awsclient\/clienterror\/waiterfailed","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError\/waiterFailed"},"doc://codes.soto.soto-core/documentation/SotoCore":{"abstract":[{"text":"The core framework for Soto, a Swift SDK for AWS","type":"text"}],"kind":"symbol","url":"\/documentation\/sotocore","type":"topic","role":"collection","title":"SotoCore","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSClient/ClientError":{"role":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ClientError"}],"title":"AWSClient.ClientError","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSClient\/ClientError","kind":"symbol","url":"\/documentation\/sotocore\/awsclient\/clienterror","abstract":[{"text":"Errors returned by ","type":"text"},{"type":"codeVoice","code":"AWSClient"},{"type":"text","text":" code"}],"fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"ClientError","kind":"identifier"}]}}}